web/angular-wallet/src/app/layout/components/burst-recipient-input/burst-recipient-input.component.scss

```diff
@@ -44,7 +44,37 @@ mat-form-field {
     word-break: normal;
     hyphenate-character: "-";
   }
+
+}
+
+.inputfile {
+  width: 0.1px;
```

Comment:
would 0px not work either?

Reply:
Per the article I grabbed this from, nope.